packages/blocks/src/api/factory.js

```diff
@@ -171,7 +187,9 @@ const getBlockTypesForPossibleFromTransforms = ( blocks ) => {
 
 			return !! findTransform(
 				fromTransforms,
-				( transform ) => isPossibleTransformForSource( transform, 'from', blocks )
+				( transform ) => {
+					return transform && isPossibleTransformForSource( transform, 'from', blocks );
```

Comment:
Can you provide or point me toward why we would be expecting a falsey transform at this point?

In removing `transform &&` from this line and the other occurrence in the file, the tests still pass, which implies one of (a) this is unnecessary or (b) the test coverage is lacking.

https://github.com/WordPress/gutenberg/pull/14908/files#diff-a5c77b070da74373cd1d937dee3a77eaR221

https://github.com/WordPress/gutenberg/pull/14908/files#diff-a5c77b070da74373cd1d937dee3a77eaR244

Reply:
I think it is a hang over from when o was handling the test for transform on this line directly rather than changing the isPossible function to handle this case.

I think we can safely remove.